json2html/jsonconv.py

```diff
@@ -28,25 +28,34 @@
 
 if sys.version_info[:2] < (3, 0):
     text = unicode
-    text_types = [unicode, str]
+    text_types = (unicode, str)
 else:
     text = str
-    text_types = [str]
+    text_types = (str,)
 
 class Json2Html:
-    def convert(self, json="", table_attributes='border="1"', clubbing=True, encode=False):
+    def convert(self, json="", table_attributes='border="1"', clubbing=True, encode=False, escape=True):
```

Comment:
that's a good idea, although I'd suggest to support both. as a user you could actually already do this by creating config as a dictionary and then pass it in after json using double star notation (**), i.e. `convert(json, **config)`. personally, if it's a one-off call with the same options I'd rather have it written out as keyword arguments - if the code line becomes too long I simply split it up. i.e. I would do something like

convert(
   json,
   clubbing=True,
   encode=True,
   ....
)